src/applications/ivc-champva/shared/components/fileUploads/FileUpload.jsx

```diff
@@ -33,12 +33,7 @@ export function FileFieldCustom(props) {
   }
 
   function uploadsMissing(data) {
-    return (
-      hasReq(data.applicants, true, true) ||
-      hasReq(data.applicants, false, true) ||
-      hasReq(data, true, true) ||
-      hasReq(data, false, true)
-    );
+    return hasReq(data.applicants, true, true) || hasReq(data, true, true);
```

Comment:
Since we ignore "optional" files that haven't been uploaded, slimming this check down to just focus on "required" files.